content/docs/demos/ingress_k8s_nginx.md

```diff
@@ -182,7 +182,7 @@ spec:
     name: "$nginx_ingress_service"
     namespace: "$nginx_ingress_namespace"
   - kind: AuthenticatedPrincipal
-    name: ingress-nginx.ingress.cluster.local
+    name: ingress-nginx.ingress-nginx.cluster.local
```

Comment:
This also needs to be updated in 2 other places in this file. It's even okay for this value to be whatever, as long as it's consistent throughout this file.